eng/dependabot/Packages.props

```diff
@@ -0,0 +1,32 @@
+<Project>
+  <!-- Packages in this file have versions updated periodically by Dependabot. Versions managed by Darc/Maestro should be in ..\Package.props. -->
+  <ItemGroup>
+    <PackageReference Update="System.Diagnostics.Process" Version="4.3.0" />
+    <PackageReference Update="System.IO.Compression" Version="4.3.0" />
+    <PackageReference Update="System.Runtime.Loader" Version="4.3.0" />
+    <PackageReference Update="Microsoft.Build.Framework" Version="16.10.0" />
+    <PackageReference Update="Microsoft.Build.Utilities.Core" Version="16.10.0" />
+    <PackageReference Update="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Update="Wcwidth.Sources" Version="0.5.0" />
+    <PackageReference Update="Microsoft.Extensions.Logging" Version="7.0.0-preview.5.22272.3" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Console" Version="7.0.0-preview.5.22272.3" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" Version="7.0.0-preview.5.22272.3" />
+    <PackageReference Update="NuGet.Configuration" Version="6.2.0-rc.146" />
+    <PackageReference Update="NuGet.Credentials" Version="6.2.0-rc.146" />
+    <PackageReference Update="NuGet.Protocol" Version="6.2.0-rc.146" />
```

Comment:
I think it make sense to update to latest public preview here.

Reply:
These have higher versions than the latest one in nuget.org. According to my try in my repo https://github.com/GangWang01/IntroduceDependabot/pulls, Microsoft.Extensions.Logging related packages are not changed by Dependabot. And Nuget are changed to latest stable 6.2.0. 
I think it needs to keep current versions. How do you think?

Reply:
It looks like Dependabot will not update to preview if the major.minor is not the same. 
See https://github.com/dependabot/dependabot-core/issues/1926.  
In my opinion staying on `6.0.0` should be fine.